apps/studio/src/features/editing-experience/components/PageCreateModal.tsx

```diff
@@ -82,7 +90,7 @@ export const PageCreateModal = ({
         <ModalCloseButton />
         <form onSubmit={handleSubmit(onSubmit)}>
           <ModalBody>
-            <Stack gap={'1.5em'}>
+            <Stack gap="1.5em">
```

Comment:
nit: try to use `rem` over `em`, ems will take the sizing of the parent instead of the root. This may result in weird values.